components/NavButtons/FAQ.tsx

```diff
@@ -1,8 +1,14 @@
-import { Button, Link } from "@chakra-ui/react";
+import Link from "next/link";
 import { Navigation } from "components/enum/navigation";
 
 export const FAQButton = () => (
-  <Link href={Navigation.FAQ} variant="button">
-    <Button>FAQ</Button>
-  </Link>
+    <Link href={Navigation.FAQ}>
+      <div className=" h-10 py-2 px-4 border-2 font-bold text-white border-white bg-transparent flex items-center rounded-md hover:border-yellow-200
```

Comment:
You don't want to use a div here - you want to use a `button`. [Button vs Divs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role). Button to Div makes it less accessible.